api/oauth.ts

```diff
@@ -0,0 +1,56 @@
+import { oauthApi } from '@api/instance';
+
+const kakaoOauth = {
+  getToken: async (code: string) => {
```

Comment:
컴포넌트 `KakaoOauth`가 존재해서 중복되는 것 같습니다. 아래처럼 변경하시는 건 어떠신가요
```suggestion
const kakaoOauthAPI = {
  getToken: async (code: string) => {
```